package.json

```diff
@@ -131,7 +131,7 @@
     "socket.io-client": "2.0.3",
     "sortablejs": "1.10.0",
     "underscore": "^1.7.0",
-    "ushahidi-platform-pattern-library": "4.4.9",
+    "ushahidi-platform-pattern-library": "^4.4.9",
```

Comment:
Remove the ^ here, we need to lock the pattern-library to this specific version, otherwise styling-changes that are about to be published gets published too early (for example, you may have seen the date-picker appearing in strange ways on some pages. Thats because changes to the date-picker is pending in a later minor, release of the pattern-library :) )

Reply:
Alright! I will fix this